geode-core/src/main/java/org/apache/geode/internal/cache/control/HeapMemoryMonitor.java

```diff
@@ -670,12 +673,31 @@ public void run() {
         // Not using the information given by the notification in favor
         // of constructing fresh information ourselves.
         if (!testDisableMemoryUpdates) {
+          try {
+            String notifType = notification.getType();
+            if (notifType.equals(MemoryNotificationInfo.MEMORY_THRESHOLD_EXCEEDED) ||
+                notifType.equals(MemoryNotificationInfo.MEMORY_COLLECTION_THRESHOLD_EXCEEDED)) {
+              // retrieve the memory notification information
+              CompositeData cd = (CompositeData) notification.getUserData();
+              MemoryNotificationInfo info = MemoryNotificationInfo.from(cd);
+              MemoryUsage usage = info.getUsage();
+              long usedBytes = usage.getUsed();
+              logger.info(
+                  "A tenured heap garbage collection has occurred.  New tenured heap consumption: "
+                      +
+                      usedBytes);
+            }
+          } catch (Exception e) {
+            logger.info(
+                "An Exception occureed while attempting to print out tenured heap consumption", e);
```

Comment:
Typo here, this should be "occurred". Also, it might be better to say "while attempting to log tenured..."